harness/determined/common/experimental/determined.py

```diff
@@ -404,3 +404,157 @@ def stream_trials_validation_metrics(
             trial_ids: List of trial IDs to get metrics for.
         """
         return trial._stream_validation_metrics(self._session, trial_ids)
+
+    def get_workspace_ids(
```

Comment:
does a function like this already exist? I'd be surprised if there wasn't something for this